libbeat/outputs/elasticsearch/api_test.go

```diff
@@ -1,3 +1,5 @@
+// +build unit integration
```

Comment:
we should not mix unit + integration tests in one file. Rather split into api_test.go and api_integration_test.go